static/src/stylesheets/module/facia/_icons.scss

```diff
@@ -2,16 +2,15 @@
 // =============================================================================
 
 .fc-container__header__title a {
-    &:after {
-        @include icon(arrow-blue-right, false);
-        content: '';
+    .inline-icon {
+        fill: colour(news);
         position: relative;
         height: .7em;
         width: 1em;
     }
 
-    &:hover:after {
-        @include icon(arrow-light-blue-right, false);
+    &:hover .inline-icon {
```

Comment:
pretty sure you can leave out the `.inline-icon` specification, it should just inherit?